kore/src/Kore/Step/Rule.hs

```diff
@@ -145,7 +145,7 @@ extractRewriteAxioms idxMod =
     mapMaybe extractRewriteAxiomFrom (indexedModuleAxioms idxMod)
 
 extractRewriteAxiomFrom
-    :: (Attribute.Axiom Internal.Symbol.Symbol, Verified.SentenceAxiom)
+    :: (Attribute.Axiom Internal.Symbol.Symbol Variable, Verified.SentenceAxiom)
```

Comment:
`fromSentenceAxiom` returns an `Either ... (QualifiedAxiomPattern Variable)` so I thought the `sentence` should also depend on a `Variable`